peachjam_search/engine.py

```diff
@@ -63,8 +63,10 @@ class SearchEngine:
             "content",
             "content_chunks",
             "case_summary",
```

Comment:
not sure if this will affect the excludes but you removed case_summary has been removed from the fields

Reply:
i've changed it to a list of includes